File: Programação I/URI/Cedula.java
import java.util.Scanner;

public class Cedula {
    public static void main( String args[]) {
        Scanner input = new Scanner( System.in);
        int reste;

        System.out.print("Digite um valor: ");
        int valor = input.nextInt();
        System.out.println(valor);
        System.out.println(valor / 100 + " nota(s) de R$ 100,00");
        reste = valor % 100;
        System.out.println(reste / 50 + " nota(s) de R$ 50,00");
        reste = reste % 50;
        System.out.println(reste / 20 +" nota(s) de R$ 20,00");
        reste = reste % 20;
        System.out.println(reste / 10 + " nota(s) de R$ 10,00");
        reste = reste % 10;
        System.out.println(reste / 5 + " nota(s) de R$ 5,00");
        reste = reste % 5;
        System.out.println(reste / 2 + " nota(s) de R$ 2,00");
        reste = reste % 2;
        System.out.println(reste / 1 + " nota(s) de R$ 1,00");





    }
}

File: Estrutura de Dados/Prática/Cedula.c
#include <stdio.h>
int main() {
    int valor, reste;
    printf("Digite um valor: ");
    scanf("%d" , &valor);
    printf("%d\n",valor);
    printf("%d nota(s) de R$ 100,00\n", valor / 100);
    reste = valor % 100;
    printf("%d nota(s) de R$ 50,00\n", reste / 50);
    reste = reste % 50;
    printf("%d nota(s) de R$ 20,00\n", reste / 20);
    reste = reste % 20;
    printf("%d nota(s) de R$ 10,00\n", reste / 10);
    reste = reste % 10;
    printf("%d nota(s) de R$ 5,00\n", reste / 5);
    reste = reste % 5;
    printf("%d nota(s) de R$ 2,00\n", reste / 2); 
    reste = reste % 2;
    printf("%d nota(s) de R$ 1,00\n", reste / 1); 
    return 0;
}
